### DISCUSSION TITLE
Where can we find the more detailed document for beginners?

### DISCUSSION THREAD

**Discussion by xjyang** [dcm4che/dcm4chee-arc-light#4319] [Q&A]
Where can I find a more detailed document regarding how to install it and how to use it? The content in the front page's Readme.md is ridiculously simple - please please add more words to that page, and tell beginners how/where to start! After the "Build" step, what do we get after "Build"? What's next? The "Installation" section seems irrelevant to the "Build" section.

**Comment by stevenbusher** [comment]
same here I installed the PACS add-on for Synology and need to figure out how I can archive DICOM data to it.  I am using an OPH software called HEYEX2 and need to create a simple DICOM contact to the archive and send data.  Cant figure it out and would love some documentation.
